data_architect_misc/data_transformer/transform_functions/eu_italy_transform_functions.py

```diff
@@ -0,0 +1,82 @@
+"""This is the subclass of Transform function for Italy (Eu division).
+
+We will define transform functions specific to Italy here.
+
+Author: Maicol Contreras
+Last Modified: December 2, 2020
+"""
+import pandas as pd
+from datetime import datetime as dt
+
+from constants import comp_harm_constants
+from constants.transform_constants import KEY_CURRENT_INPUT_FILE, KEY_HEADER
+from transform_functions.common_comp_harm_transform_functions import CommonCompHarmTransformFunctions
+
+
+class EuItalyTransformFunctions(CommonCompHarmTransformFunctions):
+    """
+    All custom (uncommon) transform functions **SPECIFIC to
+    individual processing task** must be defined as part
+    of this class.
+    """
+    ITALY_SPECIFIC_CATEGORY_MAPPINGS = {
+        # Add any Italy-specific mappings that cannot be used for other EU countries here.
+        "(?i).*ANTICALCARE.*": "Home Care",
+        "(?i).*GESTIONE.*CASA.*": "Home Care",
+    }
+
+    def __init__(self, config):
+        self.config = config
+        self.category_mappings = dict(
+            comp_harm_constants.ITALIAN_CATEGORY_MAPPINGS,
+            **EuItalyTransformFunctions.ITALY_SPECIFIC_CATEGORY_MAPPINGS)
+
+    def create_dataframe_pivotting_the_media_date_and_spend_values(
```

Comment:
We need to break this gigantic function into smaller pieces. Let's talk about this tomorrow morning.